AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSignedInViewCreditsActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAdobeStockSignedInViewCreditsActionGroup">
+        <click selector="{{AdobeStockSection.userNameButton}}" stepKey="openAccountMenu"/>
+        <see userInput="AVAILABLE" selector="{{AdobeStockSection.userQuota}}" stepKey="seeAvailableText"/>
```

Comment:
Could you please move and store the following texts to a data entity?

Reply:
@eduard13 Done, thanks!